packages/blade/src/_helpers/storybook/Sandbox/Sandbox.web.tsx

```diff
@@ -104,4 +104,28 @@ function Sandbox({
   );
 }
 
+export type RecipeSandboxProps = {
+  files: Record<string, string> | null;
+  dependencies: Record<string, string> | null;
+};
+
+export const RecipeSandbox = ({ files, dependencies }: RecipeSandboxProps): JSX.Element => {
+  return (
+    <Sandpack
+      template="react-ts"
+      files={files}
+      customSetup={{
+        dependencies: {
+          ...dependencies,
+          '@razorpay/blade': '*',
```

Comment:
> wouldn't you ci version git sha thing come here as well once that PR is merged? so in recipes as well you can open the recipe with the latest commit for blade?

Yup will add that now that the other PR is merged. 

> why do we need this sandbox and the other sandbox both?

Just different usecases because in `Sandbox` we want to have simple API where we can define one file. So we just do 
```tsx
<Sandbox>
{`
  Your code
`}
</Sandbox>
```

but here we want different API where we can pass files and dependencies.

We can add lots of checks in first sandbox and make it work for both but that just seems a bit more complex than this one. Also it will mean we make our types less strict.

Lemme share some of the code between both components though.